package.json

```diff
@@ -42,7 +42,7 @@
     "mxgraph": "3.7.2-2",
     "noty": "^3.1.2",
     "randomcolor": "^0.5.3",
-    "syncmeta-plugin": "git+https://github.com/rwth-acis/syncmeta-plugin.git",
+    "syncmeta-plugin": "git+https://github.com/rwth-acis/syncmeta-plugin.git#ifml",
```

Comment:
@patricia-patricia Can we merge the changes in syncmeta-plugin?

Reply:
Yes, we can remove the branch name.

Reply:
resolved